<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_396">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_233">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_386">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="txtTagListFilter"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="lstTagSessionFilter">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkShowDisplayable">
         <property name="text">
          <string>Show only Displayable Items</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkShowFavorites">
         <property name="text">
          <string>Show only Favorites</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkFilterProjectRelated">
         <property name="text">
          <string>Filter Related Items</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdAddFavorite">
         <property name="text">
          <string>Mark Table as Favorite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="lstTagTables">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_234">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget_29">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_134">
          <attribute name="title">
           <string>List Elements</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_387">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_235">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_388">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="txtNewTagPhraseName"/>
               </item>
               <item>
                <widget class="QPushButton" name="cmdAddNewPhraseName">
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>ADD</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="txtMstrSelected_id"/>
             </item>
             <item>
              <widget class="QLineEdit" name="txtSQLStatement"/>
             </item>
             <item>
              <widget class="QListView" name="lstTags">
               <property name="minimumSize">
                <size>
                 <width>175</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::MultiSelection</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="cmdAddSelectedTag">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Add Selected Tag</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="cmdRemoveTag">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Remove</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="cmdRemoveNoneTable">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Remove None Table</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="cmdRemoveNoneRecord">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Remove None Record</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableView" name="tbltaglist">
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_136">
          <attribute name="title">
           <string>Tree Elements</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_389">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_236">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_390">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="lstTagTreeFilter_4"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_391">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_313">
                 <property name="text">
                  <string>Primary Key</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_46"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_392">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_314">
                 <property name="text">
                  <string>Secondary Key</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_47"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTreeView" name="tvForTag_3"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
